manifestival.go

```diff
@@ -207,8 +207,10 @@ func annotate(spec *unstructured.Unstructured, key string, value string) {
 func lastApplied(obj *unstructured.Unstructured) string {
 	ann := obj.GetAnnotations()
 	if len(ann) > 0 {
-		delete(ann, v1.LastAppliedConfigAnnotation)
-		obj.SetAnnotations(ann)
+		if _, ok := ann[v1.LastAppliedConfigAnnotation]; ok {
+			delete(ann, v1.LastAppliedConfigAnnotation)
+			obj.SetAnnotations(ann)
+   		}
```

Comment:
Shouldn't be needed with the above change

Reply:
Removed